src/styles/blocks/header.scss

```diff
@@ -0,0 +1,82 @@
+@import '../utils/variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  padding: 0 50px;
+
+  &-logo {
+    height: 40px;
+    width: 40px;
+  }
```

Comment:
The `.header-logo` class seems to be an element of the `.header` block. It should be named `.header__logo` to comply with the BEM naming convention.